src/app/features/courses/auth/login/login.component.html

```diff
@@ -1,11 +1,30 @@
-<div class="login__content">
```

Comment:
Why did you add login component course instead of adding it to 'features/authorization' module?